pkg/result/processors/sort_results_test.go

```diff
@@ -188,7 +184,7 @@ func TestCompareNested(t *testing.T) {
 }
 
 func TestNumericCompare(t *testing.T) {
-	var tests = []struct {
+	tests := []struct {
```

Comment:
This can be fixed automatically by the `gofumpt` linter. I created the PR https://github.com/golangci/golangci-lint/pull/4500 to enable it